builder/base/kss_builder_base.js

```diff
@@ -155,6 +155,35 @@ class KssBuilderBase {
     });
   }
 
+
+  /**
+   * Resolve the builder path from the given file path.
+   *
+   * Call this static method to resolve the builder path.
+   *
+   * @param {string} builder The path to a builder or a builder
+   *    to load.
+   * @returns {string} resolved path
+   */
+  static builderResolve(builder) {
+    const cmdDir = process.cwd();
+    const kssDir = path.resolve(__dirname, '../..');
+    const pathsToResolve = [
+      cmdDir, // looking from commande path
+      path.resolve(cmdDir, 'node_modules'), // looking for external module
+      kssDir, // kss native builder
+      path.resolve(kssDir, 'node_modules') // old npm version
+    ];
+    let relsolvedPath = builder;
```

Comment:
Shouldn’t this be named `resolvedPath` In stead of `relsolvedPath`

Reply:
well seen